packages/core/src/components/overlay/overlay.tsx

```diff
@@ -319,11 +335,11 @@ export class Overlay extends AbstractPureComponent2<OverlayProps, IOverlayState>
             if (isFocusOutsideModal) {
                 // element marked autofocus has higher priority than the other clowns
                 const autofocusElement = this.containerElement.querySelector("[autofocus]") as HTMLElement;
```

Comment:
I noticed that `InputGroup`'s `autoFocus` prop is passed to the `input` element as is. React doesn't actually place the autofocus attribute (see https://github.com/facebook/react/issues/11851#issuecomment-351672131), so this querySelector is unlikely to work unless the consumer explicitly sets the DOM `autofocus` (no caps) attribute instead. Not sure if we want to do anything about this though, since React will warn if you set the DOM attribute directly

Reply:
we should do something about this in a FLUP. not a high priority though

Reply:
Filed https://github.com/palantir/blueprint/issues/4908